PullRequestCreated/PullRequestCreated.ts

```diff
@@ -38,6 +38,23 @@ class PullRequestCreated extends OctokitAction {
     }
   }
 
+  private additionalFields(parameters: IssueParameters): any {
+    let addutinalFields = {};
+    if (parameters?.parent) {
+      addutinalFields = { parent: { key: parameters.parent } };
+    }
+    const inputAdditionFields = this.getInput('additional-fields');
+    if (inputAdditionFields) {
+      try {
+        const fields = JSON.parse(inputAdditionFields);
+        addutinalFields = { ...addutinalFields, ...fields };
```

Comment:
This should not be inside the `try`. What about extracting `parseAdditionalFields` that would return the object, or null? And here, we would use the resulting value
```suggestion
```

Reply:
..or empty object `{ }` instead of null, after updatin the other comment about `...` usage